src/nvim/file_search.c

```diff
@@ -749,7 +749,7 @@ char_u *vim_findfile(void *search_ctx_arg)
          * Expand wildcards like "*" and "$VAR".
          * If the path is a URL don't try this.
          */
-        if (path_with_url(dirptrs[0])) {
+        if (path_with_url((char *)dirptrs[0])) {
           stackp->ffs_filearray = (char_u **)xmalloc(sizeof(char *));
```

Comment:
I'm willing to be overridden on this, but I think that's outside the scope of this PR.  If we really want to have it done in this I can do it as a "passing by" type reference though.